iaso/tests/api/test_storage.py

```diff
@@ -423,15 +425,15 @@ def test_list_base(self):
         response = self.client.get("/api/storage/")
         self.assertEqual(response.status_code, 200)
         received_json = response.json()
-        self.assertEqual(
+        self.assertListEqual(
             received_json,
             [
                 {
                     "updated_at": 1580608922.0,
                     "created_at": 1580608922.0,
                     "storage_id": "EXISTING_STORAGE",
                     "storage_type": "NFC",
-                    "status": {"status": "OK", "reason": "", "comment": "", "updated_at": None},
+                    "storage_status": {"status": "OK", "reason": "", "comment": "", "updated_at": None},
```

Comment:
I believe that when status is "OK", the reason and comment should be null. Updated at should be set